erica/erica_legacy/request_processing/erica_input/v2/grundsteuer_input_grundstueck.py

```diff
@@ -0,0 +1,86 @@
+from enum import Enum
+from typing import Optional, List, Literal
+
+from pydantic import root_validator, validator, constr
+
+from erica.erica_legacy.request_processing.erica_input.v2.camel_case_input import CamelCaseInput
+
+
+class Grundstuecksart(str, Enum):
+    einfamilienhaus = "einfamilienhaus"
+    zweifamilienhaus = "zweifamilienhaus"
+    wohnungseigentum = "wohnungseigentum"
+    baureif = "baureif"
+    abweichende_entwicklung = "abweichendeEntwicklung"
+
+    def is_bebaut(self):
+        return self in [Grundstuecksart.einfamilienhaus, Grundstuecksart.zweifamilienhaus,
+                        Grundstuecksart.wohnungseigentum]
+
+
+class Adresse(CamelCaseInput):
+    # string with 1-4 digits followed by an alphanumeric string of any length
+    hausnummer: Optional[constr(regex=r"([0-9]{1,4})([a-zA-Z0-9]*)")]  # noqa: F722
+    strasse: Optional[str]
+    zusatzangaben: Optional[str]
+    plz: Optional[str]
+    ort: Optional[str]
+
+    @validator("strasse", always=True)
+    def strasse_mus_be_present_if_hausnummer_given(cls, v, values):
```

Comment:
@shrzaf Do you remember why we included this here? The rule in the Jahresdokumentation has been removed and we don't specifically rely on this condition to be checked, correct? So I'd just delete this? Otherwise we'll get errors as we don't check this in the webapp.

Reply:
Has it? AFAIK the rule is still there in the latest version on drive, look for GW1_006

Reply:
Turns out it is indeed removed in version 35.5.4.0. So definitely go ahead and delete it. And thanks!